### DISCUSSION TITLE
Which language are you using for backend development?

### DISCUSSION THREAD

**Discussion by valentinajemuovic** [techexcellenceio/techexcellence#2] [Polls]

**Comment by YahyaQandel** [comment]
I think you might think of adding  Golang to poll  as it is well known and used now.

**Reply by valentinajemuovic** [reply]
Thanks @YahyaQandel, what are your perspectives regarding Golang, what are the key advantages?

**Reply by YahyaQandel** [reply]
-  Golang sort of similar to how we used to write c++ and structs but it is not object oriented and has no classes only struct.
- Deploying go project is much more easier than most of other technologies as it only build a binary file and you can deploy it anywhere easily.
- testing is already implicit library on go doesn't require any additional installations and the convention in writing tests in golang is really helpful and fast called 'table driven'
- go is one of the fastest stacks in terms or request response in addition to its  optimization in background tasks using  goroutines.

@valentinacupac

**Comment by marcus-sa** [comment]
TypeScript as language and https://deepkit.io as framework.

**Reply by valentinajemuovic** [reply]
@marcus-sa, what are the key advantages of DeepKit compared to other frameworks or not using a TS framework? Also, what kind of development do you do with TypeScript, is it backend, frontend, both?

**Reply by marcus-sa** [reply]
@valentinacupac 
I do everything with TypeScript.
Backend, frontend, infrastructure, etc.

There are many key advantages to Deepkit compared to alternatives.
Some of them are:
- Fastest [runtime type system](https://deepkit.io/library/type)
- Fastest [ORM](https://deepkit.io/library/orm)
- Fastest [RPC](https://deepkit.io/library/rpc)
- Best developer experience for any TS framework.

And a lot more.

**Comment by kharaone** [comment]
I'm doing both C# and Typescript, depending on the projects. Maybe changing the poll to allow multiple choices
